urbackupserver/www2/src/features/settings/LdapAd/ldapAdForm.tsx

```diff
@@ -0,0 +1,126 @@
+import { z } from "zod/v4-mini";
+
+import { integerValidation, VALIDATION_MESSAGES } from "../Form/validation";
+import type { LdapSettingsVals } from "../../../api/urbackupserver";
+import type { BaseField, Field } from "../Form/types";
+
+export type LdapSettingsKey = keyof LdapSettingsVals;
+
+export const LDAP_LABELS: Record<LdapSettingsKey, string> = {
+  ldap_login_enabled: "Enable logins via LDAP/AD",
```

Comment:
Perhaps we can directly use the labels with lingui and not put the english translations here. But that would be another pass anyway